pom.xml

```diff
@@ -21,6 +21,7 @@
 
         <version.groovy>2.4.20</version.groovy>
         <version.feel>1.12.3</version.feel>
+        <version.kotlin>1.3.72</version.kotlin>
```

Comment:
The Kotlin version seems to be a bit outdated. Can we use `1.4.10` instead?

Reply:
This is a bit complicated, because all Kotlin artifacts using bom from Spring Boot, but I think I figured it out. I will make a commit shortly

Reply:
Done, can you take a look, please? :)